services/truemetrics/TruemetricsProvider.android.tsx

```diff
@@ -22,77 +34,81 @@ const TruemetricsProvider = (props: Props) => {
       return
     }
 
-    const isAlreadyRecording = ExpoTruemetricsSdk.isRecordingInProgress()
-    if (isAlreadyRecording) {
-      return
-    }
-
-    ExpoTruemetricsSdk.startRecording()
+    // const isAlreadyRecording = ExpoTruemetricsSdk.isRecordingInProgress()
+    // if (isAlreadyRecording) {
+    //   return
+    // }
 
-    const recordingStarted = ExpoTruemetricsSdk.isRecordingInProgress()
-    if (recordingStarted) {
-      console.log("Truemetrics: started recording")
-    } else {
-      console.log("Truemetrics: failed to start recording")
-    }
-  }
+    // ExpoTruemetricsSdk.startRecording()
 
-  const stopRecording = () => {
-    ExpoTruemetricsSdk.stopRecording()
-    const recordingInProgress = ExpoTruemetricsSdk.isRecordingInProgress()
-    if (recordingInProgress) {
-      console.log("Truemetrics: failed to stop recording")
-    } else {
-      console.log("Truemetrics: stopped recording")
-    }
+    // const recordingStarted = ExpoTruemetricsSdk.isRecordingInProgress()
+    // if (recordingStarted) {
+    //   console.log("Truemetrics: started recording")
+    // } else {
+    //   console.log("Truemetrics: failed to start recording")
+    // }
   }
 
   const initializeIfNeeded = useCallback(() => {
+
+    // This is the line that causes the crash.
     const alreadyInitialized = ExpoTruemetricsSdk.isInitialized()
```

Comment:
If you replace this line with something inert (like `const alreadyInitialized = true`) it runs ok, which makes me confident that the issue is coming from this line